manual/python/index.md

```diff
@@ -60,9 +60,87 @@ caption="Example of graphics generated with PyROOT."
 
 > **Note that** the code above can also be written in a script file and executed with Python. In that case, the script will run to completion and the Python process will terminate, making the created canvas disappear. If you want to keep the Python process alive and thus be able to inspect your canvas, run the script with `python -i script_name.py`.
 
-## Interface
+## User interface
 
-*Content is coming soon!*
+Besides being the entry point to all ROOT functionality, the `ROOT` Python module also offers an interface to configure PyROOT and to manipulate PyROOT objects at a lower level. The next subsections describe in more detail the characteristics of the PyROOT user interface.
+
+### Configuration options
+
+After importing the ROOT module, one can access the PyROOT configuration object as `ROOT.PyConfig`. Such object has a set of properties that can be modified to steer the behaviour of PyROOT. For the configuration to be taken into account, it needs to be applied right after `ROOT` is imported. For example:
+
+```python
+import ROOT
+ROOT.PyConfig.OptionName1 = SomeValue1
+ROOT.PyConfig.OptionName2 = SomeValue2
+```
+
+The available configuration options are described next:
+
+- `DisableRootLogon` (default `False`): just like it happens in C++, PyROOT also supports the [rootlogon](https://root.cern/doc/master/rootlogon_8C.html) functionality. When PyROOT starts, it will look for a file called `.rootlogon.py` in the user's home directory and, if such file exists, PyROOT will import it. This can be used to make some settings every time PyROOT is launched, for example defining some style for your plots:
```

Comment:
If I understand correclty `rootlogon.py` is executed at `import ROOT` time (I guess that's what "when PyROOT starts" means)? If so, how can a change of setting that is performed _after_ ROOT is imported change this behavior?

Reply:
There are actually two phases in the PyROOT startup. `import ROOT`only does a few settings, but most of the initialization is done lazily at first lookup (e.g. when you do `ROOT.TTree`). That is why if you configure options after just `import ROOT` it is fine, you are still on time. I can change `When PyROOT starts` by `During PyROOT initialization` if you think that is more clear, but I would not mention the two phases here, I think it will confuse more than help.

Reply:
ah, I see!

> I think it will confuse more than help.

imho, without mentioning that rootlogon.py is executed upon first lookup and not at `import ROOT` time, this section is very confusing. "during PyROOT initialization" has the same meaning as "when PyROOT starts" for anyone who does not know about the two phases (e.g. me or anyone but you, Stefan, Wim, maybe Axel :D )

Reply:
I think I explain what the user needs to know: that the options need to be set right after `import ROOT`. The reason why, I don't think its crucial that they know. I would prefer not to go into details about lookups to keep the explanation as simple as possible. In any case, what difference does it make for a user (who knows the configuration options must be set after `import ROOT`) if the real configuration happens at first lookup? Pretty much anything the user will do after `import ROOT` is a lookup anyway.

Reply:
as you prefer